### PR TITLE
Add Hangman game

### PR DESCRIPTION
**Pull request by MahraibFatima**
## PR Description

Create Hangman game in python for beginners.

#### Features:

- Randomly generated words for each game.
- Visual representation of the Hangman.
- User-friendly interface.
- Guess letters to uncover the hidden word.
- Have a limited number of attempts to guess the word correctly.

### Checklist

-  [x]  I have gone through the [contributing guide](https://github.com/animator/learn-python/blob/main/CONTRIBUTING.md)
-  [x]  I have updated my branch and synced it with project `main` branch before making this PR

## Undertaking

I declare that: 

1. The content I am submitting is original and has not been plagiarized.  
2. No portion of the work has been copied from any other source without proper attribution.  
3. The work has been checked for plagiarism, and I assure its authenticity.  

I understand that any violation of this undertaking may have legal consequences that I will bear and could result in the withdrawal of any recognition associated with the work. 

- [x] I Agree

### CODE REVIEW

**Comment by animator** [comment]
You have modified the PR template. You should mark the checkboxes properly:

`- [x]` instead of `- ✅`

**Comment by MahraibFatima** [comment]
@animator kindly re-check again.

File: contrib/mini-projects/index.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # List of sections
 
-- [Section title](filename.md)
+- [Hangman game](Hangma.md)
```
Comment (animator) [inline contrib/mini-projects/index.md:3]: Incorrect filename

**Comment by animator** [comment]
File name should be lowercase

**Comment by animator** [comment]
Add comments in code to explain what is happening in each code block.

**Comment by animator** [comment]
This PR is unethical. When you are contributing to a book it is expected everything is cross-checked multiple times. You haven’t even run the code once before submitting.

**Pull request closed without merging**